### DISCUSSION TITLE
Index subwords in stars search

### DISCUSSION THREAD

**Discussion by sanxiyn** [community/community#60288] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have starred both https://github.com/djc/askama and https://github.com/mitsuhiko/minijinja.

askama's description is "Type-safe, compiled Jinja-like templates for Rust". minijinja's description is "MiniJinja is a powerful but minimal dependency template engine for Rust".

When I search stars like https://github.com/stars?q=jinja, askama matches, but minijinja doesn't. I want minijinja to match.

There could be many ways to fix this, for example repository name could match substrings. But one thing that can be improved is to index "MiniJinja" and similarly cased words to all of "mini", "jinja", and "minijinja".

**Comment by moonfury-ops** [comment]
Thank you

**Reply by Joey3120a** [reply]
Yo!

**Reply by porterjb** [reply]
Thanks

**Comment by aboahmd1** [comment]
https://youtube.com/@user-zs3kc8kf2c?si=86ZbnuGt_eXN-ohK

**Reply by BezRegi** [reply]
1
